newm-server/src/main/kotlin/io/newm/server/features/song/SongRoutes.kt

```diff
@@ -86,29 +86,29 @@ fun Routing.createSongRoutes() {
                             )
```

Comment:
It turns out that with my previous routing refactoring PR, I broke exactly the only 2 routes that are not unit tested. Lol.